pages/page.py

```diff
@@ -33,11 +32,7 @@ def __init__(self, testsetup):
 
     @property
     def is_the_current_page(self):
-        if self._page_title:
-            page_title = self.page_title
-            Assert.equal(page_title, self._page_title,
-                         "Expected page title: %s. Actual page title: %s" %
-                         (self._page_title, page_title))
+        return self._page_title == self.page_title
```

Comment:
Is that `if` in the original totally unnecessary? Will every page object have `self._page_title` defined? I am assuming so, but am also wondering why that `if` was there in the first place.